maintenance_request_report/views/report_maintenance_request.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 - TODAY, Marcel Savegnago - Escodoo
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <data>
+        <template id="report_maintenance_request_document">
+            <t t-call="web.external_layout">
```

Comment:
I think that maintenance requests are usually internal activities, so, we should use internal_layout. What do you think?

Reply:
I agree